packages/client/src/tx/types.ts

```diff
@@ -89,14 +142,57 @@ export type Transaction<
   Name extends string,
   Asset,
 > = {
+  /**
+   * Pack the transaction, sends it to the signer, and return the signature
+   * asynchronously. If the signer fails (or the user cancels the signature)
+   * it'll throw an error.
+   *
+   * @param from       `PolkadotSigner`-compliant signer.
+   * @param txOptions  Optionally pass any number of txOptions.
+   * @returns Encoded `SignedExtrinsic` ready for broadcasting.
+   */
   sign: TxSignFn<Asset>
+  /**
+   * Observable-based all-in-one transaction submitting. It'll sign, broadcast,
+   * and track the transaction. The observable is singlecast, i.e. it'll sign,
+   * broadcast, etc at every subscription. It'll complete once the transaction
+   * is found in a `finalizedBlock`.
```

Comment:
```suggestion
   * Observable-based all-in-one transaction submitting. It will sign, broadcast,
   * and track the transaction. The observable is singlecast, i.e. it'll sign,
   * broadcast, etc at every subscription. It will complete once the transaction
   * is found in a `finalizedBlock`.
```

It's alright, and def up for discussion based on personal preference, but I think it reads better without the contraction